packages/office-ui-fabric-react/src/utilities/dateMath/DateMath.ts

```diff
@@ -198,4 +198,73 @@ function getDatePartHashValue(date: Date) {
   /* tslint:disable:no-bitwise */
   return date.getDate() + (date.getMonth() << 5) + (date.getFullYear() << 9);
   /* tslint:enable:no-bitwise */
-}
\ No newline at end of file
+}
+
+/**
+ * Returns the week numbers for each week in a month.
```

Comment:
>Returns the week numbers for each week in a month. [](start = 3, length = 50)

Please follow the correct formatting for the comments following the standard in the rest of the file